frontend/src/Explorer/View/Common.purs

```diff
@@ -158,7 +160,10 @@ txBodyView (TxBodyViewProps props) =
             <<< map txFromView $ props ^. txbInputs
         , P.div
             [ P.className "to-hash-container bg-transaction-arrow" ]
-            <<< map txToView $ props ^. txbOutputs
+            [ P.div
+                [ P.className "to-hash-wrapper"]
+                <<< map txToView $ props ^. txbOutputs
```

Comment:
Are we somewhere getting advantage of using lenses ? Because record accessor seems more readable and I don't see a benefit of using lenses here. It adds unnecessary complexity, sintax is more complex, more to type and we don't have any pros of using it here (or anywhere maybe?)